src/components/MemberEditor/index.js

```diff
@@ -41,7 +42,16 @@ class MemberEditor extends Component {
 
     constructor(props) {
         super(props);
-        this.state = {};
+        this.state = {
+            toBack: false
```

Comment:
este flag no es parte del `render` sino de un estado del componente. Si no afecta el render en lugar de hacer un `this.setState(...)` usar directamente `this.goingBack = true`

Reply:
Listo